frontend/src/components/AccountCreationContainer.vue

```diff
@@ -0,0 +1,129 @@
+<template>
```

Comment:
Maybe we could look into combining some code of this and the EditContainer

Reply:
we could may capsule the html code again, i dont see, where we could capsule the ts code.

Reply:
We could distinguish inside the EditContainer if the supplied user is null or not, and if it is null, we have the "create" button, if it's not null, we have the "save" and "delete" button

Reply:
Although for the sake of getting this done and merged, I'd probably leave them seperated for now